Sprint-1/stretch/aoc-2018-day1/solution.js

```diff
@@ -0,0 +1,14 @@
+const frequencyFluctuation = (changes) => {
```

Comment:
This exercise expects you to also write JS code to read the numbers from the file `input.txt` directly.